app/scripts/templates/complete_account_unlock.mustache

```diff
@@ -0,0 +1,37 @@
+<div id="main-content">
+  {{#isLinkExpired}}
```

Comment:
This path should not be possible. If a link is not in the database, [we silently succeed](https://github.com/mozilla/fxa-auth-server/pull/867/files#diff-43da5bcdd83eb1d30aa0b5e9e7473834R641).